yt/data_objects/tests/test_image_array.py

```diff
@@ -117,8 +117,9 @@ def test_image_array_rgba_png(self):
             warnings.simplefilter("always")
             # Trigger a warning.
             im_arr.write_png("clipped.png", clip_ratio=0.5)
-            assert str(w[0].message) == (
-                "'clip_ratio' keyword is deprecated. Use 'sigma_clip' instead"
+            assert str(w[0].message).startswith(
+                "The 'clip_ratio' keyword argument is a deprecated alias for 'sigma_clip'. "
+                "Please use 'sigma_clip' directly."
```

Comment:
Should you add the two lines 
since = "3.3"
removal = "4.2" 
here?

Reply:
Well here I'm matching only the beginning of the deprecation message because I'd like to avoid matching the rest of it, which is controlled by the common function `issue_deprecation_warning` ("since" and "removal" in particular are formatted by that function).